data/project/ember/beta.md

```diff
@@ -6,12 +6,12 @@ filter:
   - /ember-template-compiler/
 repo: emberjs/ember.js
 initialVersion: 3.23.0 # Manually update, see https://libraries.io/npm/ember-source throughout
-lastRelease: 3.24.0-beta.1 # Manually update
-futureVersion: 3.24.0-beta.2 # Manually update
+lastRelease: 3.24.0-beta.3 # Manually update
+futureVersion: 3.24.0-beta.4 # Manually update
 finalVersion: 3.24.0 # Manually update
 channel: beta
 cycleEstimatedFinishDate: 2020-12-28 12:00:00 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
-date: 2020-11-17 # Manually update, get date for `lastRelease`
+date: 2020-11-17 # Manually update, get date for `initialVersion`
```

Comment:
Through Percy snapshot, I realized that the date is meant to be the release date of `initialVersion`, and not `lastRelease`.

The template for `<ReleaseTimeline>` component supports this: 

https://github.com/ember-learn/ember-website/blob/4ec74256abb2f64bc1d7838120c78c37ca009f8e/app/templates/components/release-timeline.hbs#L4-L5